handbook/engineering/campaigns/goals.md

```diff
@@ -1,66 +1,59 @@
+* Users can search, filter and sort changesets by check and review status, title, repository and branch.
+* A campaign owner can nudge developers to merge changesets.
+* Campaigns can leverage the automerge capabilities provided by their code host.
 
-Our users use more than just their code host and Sourcegraph. They use ticket trackers, review systems, and time trackers.
+### Outcomes
 
-We want to discover what external systems our users want to use campaigns with, and ensure that we can integrate with them. For example, an organization that uses JIRA will likely want to be able to link tickets to campaigns and have the state updated as the campaign is executed.
-
-## Roadmap
-
-1. **Customer outreach to improve adoption of campaigns**
-1.  ~~User credentials ([RFC 242](https://docs.google.com/document/d/1SqoWWm1xs82QibrWwYsXmpmgweN6EpcKt1qXrRBjjlU/edit)), which will allow non-site-admins to create campaigns~~ (landed 3.22)
-1. Better burndown charts 
-1. Allow multiple changeset in a single repository
-1. Improve previews on updates to show the delta between existing changesets and the changesets after after the campaign spec changes are applied
-1. Allow campaign to specify target branch
-1. Create campaign spec even if subset of repos have failures
-1. Versioning/releasing of src-cli with respect to sg/sg
-1. Improved documentation of src-cli login process
-1. Add filtering/searching to campaign and changeset lists
-1. Ability to auto-merge changesets, depending on various conditions
-1. Ability to add default reviewers to changesets
-
-### Roadmap Process
-
-Our roadmapping process is a team effort. As we receive customer feedback, and as we come up with cool new features, we record these in one of three places. Customer feedback goes into our [team Productboard page](https://sourcegraph.productboard.com/feature-board/2104383-campaigns). Well-defined tasks live in our [team backlog](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fcampaigns+milestone%3ABacklog). And larger, less defined ideas live in the bottom section of our [private roadmap doc](https://docs.google.com/document/d/1zRTfK6mENKicfLwDaWgLk1dBvQVKDg-J7pwjGg8tpps/edit#), below the _fog of war_ line.
-
-It is the job of the EM and PM to pull/formulate/define tasks from the above sources and prioritize them. That prioritized list is under the [Future heading](https://docs.google.com/document/d/1zRTfK6mENKicfLwDaWgLk1dBvQVKDg-J7pwjGg8tpps/edit#heading=h.jk3gp8lyopke) of our roadmap doc.
-
-For our sprint planning, the team looks over the prioritized list, revises estimates on work needed, and agrees on these priorities. Once we are aligned, we pull as much as will comfortably fit into the next sprint, creating GitHub issues as needed, and track those in that sprint's tracking issue.
+* An increase in the average number of merged changesets is observed.
+* A reduction in the average amount of time to close a campaign is observed.
```

Comment:
I think this goal is very vague and might not be hit if customers start creating more _complex_ campaigns, which isn't a bad thing though.

Reply:
For a campaign of complexity X (however that is measured, as it depends on the size of the campaign as well as all sorts of factors related to the customer's org), the expected outcome is a reduction in the average amount of time to close a campaign.

What's a better way to say that?

Reply:
Since we don't really have any influence on the duration of a campaign, and we can't measure how long the same process would've taken without the use of campaigns, so.. is this even measurable? 
If no, how about we go with something like "the amount of support requests required to close a campaign" (not necessarily bugs, but questions and stuff included)

Reply:
We'd better have some influence on the duration of a campaign! But we can't control it, only influence it. And you are right that we can't A/B test a single campaign like that.

Reduction of support requests would be great, but it's really not the same thing. The whole value proposition of campaigns is that it saves a ton of time. It's hard to measure how much time it saves, but I still think we should try. And once we have enough customers running enough campaigns, I think we should expect to see time-saving features reduce the average amount of time to close a campaign.

Putting it another way: if we _don't_ see that reduction (when we get to that scale), I think it's a sign that something isn't right.

Reply:
Valid concern Erik, but as this is our customer's concerns, we felt like we should register it as our concern. We should work with analytics to try and find a fitting measurement.